docs/features/analysis.md

```diff
@@ -804,6 +804,36 @@ spec:
         limit: 20
 ```
 
+### Measurements Retention for Experiments
+
+If an experiment wants to retain more results of its analysis metrics, it simply needs to specify the 
+`measurementRetention` field under its specs. In the following example, all the metrics from `analyze-job` matching the 
+RegEx rule `test.*` will have their latest twenty measurements get retained instead of the default ten.
```

Comment:
qq: if an measurementRetention is defined both in analysistemplate and an experiment that refers that analysis template, which one takes priority?

Reply:
It'll be merged just like other fields (args etc.) and then validated (no duplicates etc.)

Reply:
@huikang It will throw an exception as while merging "test.*" will get shadowed by ".*" defined in the analysis template.
Basically, the metric name RegEx should be unique for the definitions to get merged or otherwise it'll throw an exception.